packages/calcite-design-tokens/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to calcite-design-tokens
+
+You want to contribute? Nice! Below are some guidelines for ensuring that your contribution makes sense for everybody.
+
+## Reporting Issues
+
+Found a problem? Want a new feature?
+
+- See if your issue or idea has [already been reported](issues).
+- Provide detailed reproduction instructions as well as what behavior is expected.
+
+## Contributing to tokens
+
+### Read [<root>/CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+These docs include initial setup, commit message format guidelines and more.
+
+### Project scripts
+
+#### Build
+
+Build the tokens to platform assets
```

Comment:
Can we use proper grammar in our docs, e.g. either append a `:` or use a complete sentence with a period.